src/keys.c

```diff
@@ -371,7 +373,10 @@ load_keys(const char* jwkdir)
             filepath[sizeof(filepath) - 1] = '\0';
             json_auto_t* json = json_load_file(filepath, 0, NULL);
             if (!json) {
-                fprintf(stderr, "Invalid JSON file (%s); skipping\n", filepath);
+                enum json_error_code code = json_error_code(&error);
+                fprintf(stderr, "Cannot load JSON file (%s); skipping\n", filepath);
+                fprintf(stderr, "error code %d text %s line %d col %d pos %d\n",
```

Comment:
For better readability, could you please add some separator here between the code, text, etc? A comma would suffice, I guess.

Reply:
Now like this:

`
tangd[18555]: Cannot load JSON file (/var/db/tang/qqq-exc-2022-05-29.jwk); skipping
tangd[18555]: error text unable to open /var/db/tang/qqq-exc-2022-05-29.jwk: Permission denied, line -1, col -1, pos 0
`